docker/dev/install.sh

```diff
@@ -122,12 +126,9 @@ echo '<settings>
 
 
 echo "Install Java linter ..."
-curl -sLJ \
-     "https://github.com/google/google-java-format/releases/download/google-java-format-1.6/google-java-format-1.6-all-deps.jar" \
-     -o /usr/local/bin/google-java-format-1.6-all-deps.jar
-curl -sLJ \
-     "https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml" \
-     -o /usr/local/bin/google_checks.xml
-curl -sLJ \
-     "https://github.com/checkstyle/checkstyle/releases/download/checkstyle-8.29/checkstyle-8.29-all.jar" \
-     -o /usr/local/bin/checkstyle-8.29-all.jar
+axel --quiet --output /usr/local/bin/google-java-format-1.6-all-deps.jar \
+     "https://github.com/google/google-java-format/releases/download/google-java-format-1.6/google-java-format-1.6-all-deps.jar"
+axel --quiet --output /usr/local/bin/google_checks.xml \
+     "https://raw.githubusercontent.com/checkstyle/checkstyle/master/src/main/resources/google_checks.xml"
+axel --quiet --output /usr/local/bin/checkstyle-8.29-all.jar \
+     "https://github.com/checkstyle/checkstyle/releases/download/checkstyle-8.29/checkstyle-8.29-all.jar"
```

Comment:
Should we use `curl` when downloading with one URL?

Reply:
I have an ambitious plan to mirror these links on AWS S3 to accelerate the CI, because Travis CI VMs run on AWS.